ceph.spec.in

```diff
@@ -1186,10 +1187,18 @@ ${CMAKE} .. \
     -DWITH_RADOSGW_KAFKA_ENDPOINT=ON \
 %else
     -DWITH_RADOSGW_KAFKA_ENDPOINT=OFF \
+%endif
+%if 0%{with cmake_verbose_logging}
+    -DCMAKE_VERBOSE_LOGGING=ON \
```

Comment:
@smithfarm  i cannot find the cmake document / code mentioning `CMAKE_VERBOSE_LOGGING`. could you point me to any of them?

Reply:
sorry, CMAKE_VERBOSE_MAKEFILE was intended